homeassistant/components/growatt_server/sensor.py

```diff
@@ -408,6 +521,7 @@ def unit_of_measurement(self):
         """Return the unit of measurement of this entity, if any."""
         return SENSOR_TYPES[self.sensor][1]
 
+    @Throttle(MIN_TIME_BETWEEN_UPDATES)
```

Comment:
I don't think this is necesarry since the probe is already throttled, let me know if this solves some issue.

Reply:
During testing I saw multiple calls to the API in 5 seconds. After I added the throttle it was gone. I tested it again and it work as intended without the adjustment. I removed it.